spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Optimizer.scala

```diff
@@ -452,3 +481,10 @@ object Optimizer {
     }
   }
 }
+
+case class GradientClippingParams(
```

Comment:
make this a trait, and implement two subclass for l2norm and constant clipping respectively

Reply:
agree